From 1180ff151a6640182c2bf50bf429364e7b744b63 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Tue, 23 Jun 2020 19:15:37 +0200
Subject: [PATCH 133/467] ARM: dts: sun8i-a83t-tbs-a711: Add regulators to the
 accelerometer

The bosch,bma250 driver will fail to probe if the regulators are
not specified in DT, because it tries to set voltage on dummy
regulators, which fails with EINVAL.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts b/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
index 603918c62..d9281c8d3 100644
--- a/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
+++ b/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
@@ -273,6 +273,8 @@ accelerometer@18 {
 		reg = <0x18>;
 		interrupt-parent = <&pio>;
 		interrupts = <7 10 IRQ_TYPE_EDGE_RISING>; /* PH10 / EINT10 */
+		vdd-supply = <&reg_dcdc1>;
+		vddio-supply = <&reg_dcdc1>;
 	};
 
 	/* NFC (NPC 100) */
-- 
2.34.0

